plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -131,7 +143,7 @@ def is_started(self) -> bool:
 
     def start(self) -> None:
         """Run the ipfs daemon."""
-        cmd = ["ipfs", "daemon", "--offline"] if self.offline else ["ipfs", "daemon"]
```

Comment:
We'll need offline daemons from now so there's no need to specify it

Reply:
Why do we need them from now? Please expand

Reply:
> We'll need offline daemons from now so there's no need to specify it

We won't need, Sorry it was a typo